packages/react-graphql/README.md

```diff
@@ -222,7 +222,7 @@ This hook accepts two arguments:
 interface QueryHookOptions<Variables = OperationVariables> {
   ssr?: boolean;
   variables?: Variables;
-  fetchPolicy?: FetchPolicy;
+  fetchPolicy?: FetchPolicy | WatchQueryFetchPolicy;
```

Comment:
Seems like we can just use `WatchQueryFetchPolicy` since, at least based on the changelog, it is `FetchPolicy | other policies`